tests/initialise_from_shared_object/Makefile

```diff
@@ -0,0 +1,9 @@
+default: input
+
+input: clean
+	gmsh -2 src/box_2d.geo
+
+	gfortran -fPIC -shared -g -o example.so src/example.F90 
```

Comment:
Hm, I guess we need to think of something more general here. We can't rely on gfortran being available.